src/metabase/util/ssh.clj

```diff
@@ -6,12 +6,20 @@
 
 (def ^:private default-ssh-timeout 30000)
 
+(defn ssh-add-key
+  "Adds SSH private key to Jsch class."
+  [jsch key-file-name]
+  (log/debug "Adding ssh key:" key-file-name)
+  (when-not (string/blank? key-file-name) (doto ^com.jcraft.jsch.JSch jsch (.addIdentity key-file-name)))
```

Comment:
`str` required here is because `clojure.string` was required as `str` in the ns block, see: https://github.com/metabase/metabase/blob/master/src/metabase/util/ssh.clj#L2

This pull request cannot compile as is this `string/blank?` reference.